src/com/google/enterprise/adaptor/Config.java

```diff
@@ -286,6 +288,7 @@ public String compute(String rawValue) {
     addKey("adaptor.docContentTimeoutSecs", "180");
     addKey("adaptor.docHeaderTimeoutSecs", "30");
     addKey("transform.pipeline", "");
```

Comment:
I'm thinking that after this CL is submitted I'll rename "transform.pipeline" to "metadata.transform.pipeline" and also rename classes in the source code base.  For backwards compatability I'll have the code continue to accept "transform.pipeline" for "metadata.transform.pipeline", but the class names will be redone.

What do you think?